# @@ Debugging Options
verbose: 0

# @@ Hardcoded variables
num:       30                
t_plan:    3
max_drone: 2

x_lim: [-2.0, 2.0]
y_lim: [-2.0, 2.0]
z_lim: [+0.0, 2.0]

vel_max: 1.73
gravity: 9.8
f_max: 1.5    # 1.5g
f_min: 0.3    # 0.3g

# @@ CBF
gamma: 0.9

# @@ Planning Parameters
dist_stop: 0.2

# @@ Optimizer Parameters
max_iter: 5
max_time: 20

weight_goal:       +1.0E+8
weight_quad_slack: +1.0E+15
weight_smoothness: +1.0E+5

# @@ Crazyflie Configuration
a_drone: 0.0675
b_drone: 0.0675
c_drone: 0.2000

buffer: 0.05

# @@ Start-Goal Configuration
init_drone: [[-1.0, -1.0, 0.5],
             [+1.0, +1.0, 0.5],
             [+1.0, -1.0, 0.5],
             [-1.0, +1.0, 0.5],
             [+1.0, +0.0, 0.5],
             [-1.0, +0.0, 0.5],
             [+0.0, +1.0, 0.5],
             [-0.0, -1.0, 0.5]]

goal_drone: [[+1.0, +1.0, 0.5],
             [-1.0, -1.0, 0.5],
             [-1.0, +1.0, 0.5],
             [+1.0, -1.0, 0.5],
             [-1.0, +0.0, 0.5],
             [+1.0, +0.0, 0.5],
             [+0.0, -1.0, 0.5],
             [-0.0, +1.0, 0.5]]



# HARD CODED VARIABLES
# num:    30                
# t_plan: 3

# # @@ True Kinematics
# vel_max: 1.73
# gravity: 9.8
# f_max: 1.5    # 1.5g
# f_min: 0.3    # 0.3g

# max_drone: 8          
# num_drone: 8           
